templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -462,19 +462,39 @@
               {
                 "Variable": "$.wasReconciled",
                 "BooleanEquals": true,
-                "Next": "Update ingested_PS attribute in Files table"
+                "Next": "Publish reconciliation update to dr2-notifications"
               }
             ],
             "Default": "Throw Reconciler job error"
           },
+          "Publish reconciliation update to dr2-notifications": {
+            "Type": "Task",
+            "Resource": "arn:aws:states:::sns:publish",
+            "Parameters": {
+              "Message": {
+                "properties": {
+                  "messageId": "$.reconciliationSnsMessage.properties.messageId",
+                  "parentMessageId": "$.reconciliationSnsMessage.properties.parentMessageId",
+                  "timestamp": "$.reconciliationSnsMessage.properties.timestamp",
+                  "type": "preserve.digital.asset.ingest.update"
+                },
+                "parameters": {
+                  "assetId": "$.reconciliationSnsMessage.parameters.assetId",
+                  "status": "Asset has been ingested to the Preservation system."
+                }
```

Comment:
Just to confirm the mappings upstream of this...
- `$.reconciliationSnsMessage.properties.messageId` is a new UUID?
- `$.reconciliationSnsMessage.properties.parentMessageId` is the `messageId` of the message in the lock table?
- `$.reconciliationSnsMessage.properties.timestamp` is an ISO timestamp?
- `$.reconciliationSnsMessage.parameters.assetId` is the `name` of the Asset item in dr2-ingest-files?

Reply:
Yes please, we're in a bit of an odd situation here. Currently the `id` in our table is generated during our preingest lambda so only we recognise it. We have a ticket in the backlog to use the TDR UUID as the `id` but can't do that until we've changed our table's primary key. So for now we need to use the `name`.